src/hooks/useInstanceConfiguration.ts

```diff
@@ -13,6 +13,18 @@ import {
 import { appInit, updateInstanceConfiguration } from '../store/app';
 import InstanceConfigurationInterface from '../domain/InstanceConfigurationInterface';
 
+// Order an array with objects based on an string in order array
```

Comment:
From our previous discussion, I suppose this ordering function is not used and afterall and could be removed.